filament/src/materials/MaterialHelpersShading.fxh

```diff
@@ -12,6 +12,132 @@ float SignNoZero(float f) {
     return f < 0.0 ? -1.0 : 1.0;
 }
 
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+//
+// Material usage getters
+//
+// The usageFlags bitfield stores whether the material should source certain attributes from 
+// textures, how it should swizzle them, and/or alter lighting. These used to be separate uniform
+// integers but we simply ran out of the spare space and were forced to pack these together. 
+//
+// Old uniform int name	        Bit	    New query from flag bitfield
+// =====================================================================
+// useBaseColorTexture	        0	    materialParams.usageFlags & 1
+// useClearCoatNormalTexture	1	    materialParams.usageFlags & 2
+// useClearCoatRoughnessTexture	2	    materialParams.usageFlags & 4
+// useMetallicTexture	        3	    materialParams.usageFlags & 8
+// useNormalTexture	            4	    materialParams.usageFlags & 16
```

Comment:
nit: pls align the columns

Reply:
Fixed in c680e4f2bf6838ae0f7480a18558640779362379